diff --git a/configure.ac b/configure.ac
index 40067279ecce..c0e6583af8ad 100644
--- a/configure.ac
+++ b/configure.ac
@@ -162,9 +162,6 @@ AC_CHECK_HEADER(sys/fanotify.h, , [AC_MSG_ERROR(
 ["Couldn't find sys/fanotify.h...your kernel might not be new enough"] )])
 AC_CHECK_FUNCS(fexecve, [], [])
 AC_CHECK_FUNCS([gettid])
-AC_CHECK_HEADER(uthash.h, , [AC_MSG_ERROR(
-["Couldn't find uthash.h...uthash-devel is missing"] )])
-
 
 echo .
 echo Checking for required libraries
diff --git a/src/Makefile.am b/src/Makefile.am
index 05d0f8e9ddd6..65a218992152 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -5,6 +5,9 @@ AM_CPPFLAGS = \
 	-I${top_srcdir} \
 	-I${top_srcdir}/src/library
 
+AM_CPPFLAGS += \
+	-I${top_srcdir}/uthash-2.3.0/include
+
 sbin_PROGRAMS = fapolicyd fapolicyd-cli
 lib_LTLIBRARIES= libfapolicyd.la
 
diff --git a/src/library/rpm-backend.c b/src/library/rpm-backend.c
index c972874311d7..0acf7ba5b84a 100644
--- a/src/library/rpm-backend.c
+++ b/src/library/rpm-backend.c
@@ -46,7 +46,7 @@
 #include <fnmatch.h>
 #include <sys/mman.h>
 
-#include <uthash.h>
+#include "uthash.h"
 
 #include "message.h"
 #include "gcc-attributes.h"
