diff --git a/configure.ac b/configure.ac
index 8188d13..6430e1e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -112,9 +112,6 @@ AC_CHECK_HEADER(sys/fanotify.h, , [AC_MSG_ERROR(
 ["Couldn't find sys/fanotify.h...your kernel might not be new enough"] )])
 AC_CHECK_FUNCS(fexecve, [], [])
 AC_CHECK_FUNCS([gettid])
-AC_CHECK_HEADER(uthash.h, , [AC_MSG_ERROR(
-["Couldn't find uthash.h...uthash-devel is missing"] )])
-
 
 echo .
 echo Checking for required libraries
diff --git a/src/Makefile.am b/src/Makefile.am
index 25afbcd..dc308ec 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -5,6 +5,9 @@ AM_CPPFLAGS = \
 	-I${top_srcdir} \
 	-I${top_srcdir}/src/library
 
+AM_CPPFLAGS += \
+	-I${top_srcdir}/uthash-2.3.0/include
+
 sbin_PROGRAMS = fapolicyd fapolicyd-cli
 lib_LTLIBRARIES= libfapolicyd.la
 
diff --git a/src/library/rpm-backend.c b/src/library/rpm-backend.c
index 8d5aa20..6e92f10 100644
--- a/src/library/rpm-backend.c
+++ b/src/library/rpm-backend.c
@@ -33,7 +33,7 @@
 #include <rpm/rpmpgp.h>
 #include <fnmatch.h>
 
-#include <uthash.h>
+#include "uthash.h"
 
 #include "message.h"
 #include "gcc-attributes.h"
